andela/denethor-ah-frontend
# #64395483 Article Read page

## What this PR does
- Setup Article Read Page
- Includes Latest publication section on landing page

## How to test locally
- Clone repo
- Check out branch `feature/164395483/single-article-view-page`
- `npm run dev-server`
- Visit http://localhost:8080/articles/1
- Visit http://localhost:8080 - checkout the latest publication section on the landing page

## Screenshots
<img width="1303" alt="Screenshot 2019-03-11 at 9 24 49 PM" src="https://user-images.githubusercontent.com/14825863/54155374-48b81800-4444-11e9-9a33-e2a843f8bb26.png">
<img width="1433" alt="Screenshot 2019-03-11 at 9 25 28 PM" src="https://user-images.githubusercontent.com/14825863/54155376-48b81800-4444-11e9-9294-fe1bc45c7fef.png">


### [Pivotal Tracker Story]
(https://www.pivotaltracker.com/story/show/164395483)
